jest.config.js

```diff
@@ -14,5 +14,8 @@ module.exports = {
     '<rootDir>/cfgov/unprocessed/apps/.+/webpack-config\.js$',
     '<rootDir>/cfgov/unprocessed/js/routes/'
   ],
-  coverageDirectory: '<rootDir>/test/unit_test_coverage'
+  coverageDirectory: '<rootDir>/test/unit_test_coverage',
+  moduleNameMapper: {
+    '^Templates(.*)$': '<rootDir>/cfgov/unprocessed/apps/owning-a-home/templates$1'
```

Comment:
No, I had to alias the templates directory because the templates weren't loading. I'm not sure why, but I only saw the error when running Webpack in dev mode. 
 https://github.com/cfpb/cfgov-refresh/pull/3889/files#diff-b8cb70fa96d6156cf6a9424ef0687568R71

See if you get the same errors on that commit.

Reply:
Ah bummer, I'll check it out. In the meantime, is it possible we can replace `owning-a-home` with a regex group, like what you're doing with `$1`? Also, why is `'^Templates(.*)$'` cap-case?

Reply:
Oh I see why it's cap-case.